db/migrate/20200220150100_add_service_type_to_destination.rb

```diff
@@ -0,0 +1,25 @@
+class AddServiceTypeToDestination < ActiveRecord::Migration[5.2]
+  def up
+    add_column :destinations, :service_type, :string
+    add_column :destinations, :service_name, :string
+    add_column :destinations, :adobe_api_key, :string
```

Comment:
We'll not be needing to add `adobe_api_key` or `adobe_api_secret` as the columns `access_key_id` and `access_key_secret` already exist. @ckozus